toke.c

```diff
@@ -12644,22 +12653,39 @@ Perl_yyerror_pvn(pTHX_ const char *const s, STRLEN len, U32 flags)
             qerror(msg);
         }
     }
-    if (s == NULL || PL_error_count >= 10) {
-        const char * msg = "";
+    if ( s == NULL ||
+         PL_error_count >= PERL_STOP_PARSING_AFTER_N_ERRORS
+    ) {
         const char * const name = OutCopFILE(PL_curcop);
+        SV * errsv = NULL;
+        U8 raw_error_count= PERL_PARSE_ERROR_COUNT(PL_error_count);
+        U8 syntax_error= PERL_PARSE_IS_SYNTAX_ERROR(PL_error_count);
```

Comment:
U8? Surely a `bool`?

Reply:
Its a bitmap test on PL_error_count which is U8. I can convert it to bool on the fly, but there doesn't seem much point.

Reply:
Anyway, changed it to a bool.